webpack.config.js

```diff
@@ -91,6 +92,14 @@ const InteractivityConfig = {
 	...getInteractivityAPIConfig( { alias: getAlias() } ),
 };
 
+/**
+ * Config to generate the Interactivity Blocks runtime.
+ */
+const InteractivityBlocksConfig = {
+	...sharedConfig,
+	...getInteractivityBlocksConfig( { alias: getAlias() } ),
```

Comment:
It appears that we aren't passing a significant amount of arguments to this function. Considering this, perhaps we should aim to simplify our code a bit?